ui/bin/roll20.js

```diff
@@ -0,0 +1,226 @@
+(function() {
```

Comment:
This `bin` file is auto-generated on commit via the pre-commit hook.
We should also be able to do this via github actions: https://docs.github.com/en/actions/quickstart

Reply:
https://github.com/epic-power-rpg/roll20/pull/13